cmd/gcp-controller-manager/kubelet_readonly_csr_approver.go

```diff
@@ -327,3 +329,28 @@ func validateRbac(request kubeletReadonlyCSRRequest) kubeletReadonlyCSRResponse
 		message: fmt.Sprintf("user %s is allowed to create a CSR", csr.Spec.Username),
 	}
 }
+
+// validateCommonName will make sure the common name for x509 has kubelet-ro-client
+func validateCommonName(request kubeletReadonlyCSRRequest) kubeletReadonlyCSRResponse {
+	x509cr := request.x509cr
+	if x509cr == nil || len(x509cr.Subject.CommonName) == 0 {
+		return kubeletReadonlyCSRResponse{
+			result:  false,
+			err:     nil,
+			message: fmt.Sprintf("x509cr or CommonName is empty"),
+		}
+	}
+
+	if !strings.HasPrefix(x509cr.Subject.CommonName, kubeletReadonlyCRCommonName) {
+		return kubeletReadonlyCSRResponse{
+			result:  false,
+			err:     nil,
+			message: fmt.Sprintf("x509 common name not start with %s", kubeletReadonlyCRCommonName),
```

Comment:
this sentence should be updated to 

`CommonName should start with %s` or `CommonName does not start with %s`